src/css/components/panel.scss

```diff
@@ -132,54 +116,19 @@ h4 + .panel {
     width: 100%;
   }
 
-  > .panel-row {
```

Comment:
Amost all of these variations have been consolidated in to the `Row` component or into `PanelEntry` or `Card`